Examples/HelloWorld/HelloWorld.cpp

```diff
@@ -16,9 +16,7 @@
 #include "HelloRandomAlgorithm.hpp"
 #include "HelloWhiteBoardAlgorithm.hpp"
 
-int main(int argc, char* argv[]) {
-  (void)argc;
-  (void)argv;
+int main(void) {
```

Comment:
```suggestion
int main() {
```

would also be an option

Reply:
I think I prefer `main(void)`, since it fails, if one tries to call it with arguments.